packages/gatsby-cli/.babelrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    ["@babel/env", { "modules": false, "targets": { "node": "10.13.0" } }],
+    "@babel/preset-react"
+  ],
+  "plugins": ["@babel/plugin-transform-runtime"],
+  "overrides": [
+    {
+      "test": ["**/*.ts", "**/*.tsx"],
+      "plugins": [["@babel/plugin-transform-typescript", { "isTSX": true }]]
+    }
+  ]
+}
```

Comment:
Any reason for this being a json file and not a regular rc file?

Reply:
Not really. Only reason it's swapped is because I used setup from https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-recipes as boilerplate for rollup setup and it had json file 🤷

Reply:
I can move it back to rc format if you prefer it that way

Reply:
it's more in line with what our other packages have

Reply:
Cool, I'll do that then ;)